### DISCUSSION TITLE
Get https://registry-1.docker.io/v2/: EOF

### DISCUSSION THREAD

**Discussion by SebastianMuskalla** [containrrr/watchtower#670] [Q&A] [answered]
Hey,

Since about 2 weeks ago, I regulary get email notifications by watchtower of the following shape:

`2020-11-03 16:26:05 (info): Unable to update container "/watchtower": Error response from daemon: Get https://registry-1.docker.io/v2/: EOF. Proceeding to next.`

It happens for various containers that watchtower is trying to update and for various specific URLs, but the domain is always `registry-1.docker.io` and the error message is always simply `EOF`.
It doesn't always happen; sometimes, watchtower is able to sucessfully update containers.

Is this a problem of watchtower, or a problem of the docker daemon itself?
There don't seem to be any networking problems outside of this issue.

Thanks!

**Comment by piksel** [comment]
Since you are from [Germany](https://github.com/docker/for-win/issues/6192#issuecomment-629427316), maybe [this](https://github.com/docker/for-mac/issues/3493#issuecomment-616338458) is relevant?

**Reply by SebastianMuskalla** [reply]
Thanks for the reference!
This is happening on a server that is hosted in a datacenter though.
I hope that they are not using a single fritzbox to connect the server to the internet ;)

**Comment by simonaronsson** [comment]
Would you be able to narrow it down to some config we can reproduce?

**Comment by simonaronsson** [comment, marked as answer]
This is what would happen if you get rate limited by dockerhub. As this is quite old, and we've since added HEAD requests as a first comparison, I don't think this question is relevant anymore. If this is still an issue, let us know.
